os/arch/Kconfig

```diff
@@ -756,6 +756,15 @@ config RAM_REGIONx_HEAP_INDEX
 		It means that its region can be handled to the index-th heap. 
 		Index can be from 0 to (MM_NHEAPS-1).
 
+config FS_TMPFS_HEAP_INDEX
```

Comment:
Is it okay not to bind with TMPFS config?

Reply:
I will modify depends on FS_TMPFS

Reply:
This is not good option.
Not only tmpfs is ram based file system but smartfs also can be mounted based on ram(ramdisk)
Since we supports ramdisk, all file system using MTD can be mounted on ram...
mm module should provide memory region for file systems.
```
#ifdef CONFIG_MM_REGION_SYSTEM
 #define kmm_malloc(size) malloc_at(CONFIG_MM_REGION_SYSTEM, size)
 #define kmm_zalloc(size) zalloc_at(CONFIG_MM_REGION_SYSTEM, size)
 #define kmm_realloc(oldmem, size) realloc_at(CONFIG_MM_REGION_SYSTEM, oldmem, size)
#else 
 #define kmm_malloc(size) kmm_malloc(size)
 #define kmm_zalloc(size) kmm_zalloc(size)
 #define kmm_realloc(oldmem, size) kmm_realloc(oldmem, size)
#endif
```

Reply:
I will modify